package.json

```diff
@@ -30,15 +32,34 @@
   "license": "MIT",
   "devDependencies": {
     "@fastify/pre-commit": "^2.0.2",
+    "@rollup/plugin-commonjs": "^24.0.1",
+    "@rollup/plugin-inject": "^5.0.3",
+    "@rollup/plugin-node-resolve": "^15.0.1",
+    "browserify": "^17.0.0",
+    "buffer": "^6.0.3",
+    "esbuild": "^0.17.4",
+    "esbuild-plugin-alias": "^0.2.1",
+    "events": "^3.3.0",
     "ioredis": "^5.2.3",
+    "path-browserify": "^1.0.1",
+    "playwright": "^1.29.2",
+    "process": "^0.11.10",
     "proxyquire": "^2.1.3",
+    "rollup": "^3.11.0",
+    "rollup-plugin-polyfill-node": "^0.12.0",
     "snazzy": "^9.0.0",
     "standard": "^17.0.0",
+    "stream-browserify": "^3.0.0",
     "tap": "^16.3.0",
-    "tsd": "^0.25.0"
+    "tap-mocha-reporter": "^5.0.3",
+    "tap-parser": "^12.0.1",
+    "tape": "^5.6.3",
+    "tsd": "^0.25.0",
+    "vite": "^4.0.4",
+    "webpack": "^5.75.0",
+    "webpack-cli": "^5.0.1"
   },
   "dependencies": {
-    "abstract-logging": "^2.0.1",
```

Comment:
missing dep

Reply:
please check deps

Reply:
Oh, I remove it since is not compatible with browsers. I wrote a simple function to simulate the abstractLogging and I've reused it in both envs.

Should I restore it and keeping the function for browsers?

Reply:
Oh, I haven't noticed, but I prefer to keep the `abstract-logging` for browser too, I think it can work on browser too

Reply:
You're absolutely right. It's working on browsers too. Fixing it.